agent/structs/config_entry_discoverychain.go

```diff
@@ -639,6 +639,10 @@ type ServiceResolverConfigEntry struct {
 	// to this service.
 	ConnectTimeout time.Duration `json:",omitempty" alias:"connect_timeout"`
 
+	// LoadBalancer determines the load balancing policy and configuration for services
```

Comment:
Not sure if a later commit adds it, but you should probably add some json/hcl camel/snake kitchen-sink decoder tests to `command/config/...`, the agent bootstrap config entries code, and `agent/structs` packages to verify all of the round trip representations for this work.

Reply:
Missed that, fixed